### ISSUE TITLE
Brainstorm: Whatwould you like to make? 

### ISSUE THREAD

**Issue by gnimmelf** [Oslo-Haskell/Discussions#1]
E.g.
Droppable Microshop for my friends who use Wordpress.  (I hate Wordpress,  but love my friends). 
A system that lists companies and their software stacks and related products. (Can be expanded to a jobs listing service.. )

I'm biased towards a web service,  but all suggestions are welcome...

**Comment by tobiasgwaaler** [comment]
What's a droppable microshop? :P

**Comment by tobiasgwaaler** [comment]
And the other one: something like http://stackshare.io/ ?

**Comment by gnimmelf** [comment]
1. Some sort of simple wordpress replacement. Droppable in the sense that I can just copy stuff onto a server,  configure db conn and merchant account,  and that's it. Snap looks promising.. 
2. Yes, stackshare.io, and searchable by company/region + stack: all in Oslo using Nodejs, Hapi,  Mongodb. 

Those were just the ones off the top of my head since I'm using Snap to teach myself Haskell,  and for the second one, I am annoyed at finn.no's job listing for not having this,  hehe.

**Comment by tobiasgwaaler** [comment]
I have an idea I've been toying with for a while without too much progress:
Now that we have kolonial.no I want to have a bookmarklet that lets me add all the ingredients for a recipe to my shopping cart. For any recipe site! That means parsing the ingredients list, which turns out to be really difficult. Could be solved using machine learning algorithms, but that requires lots of data/training. Some kind of hybrid solution is probably the way to go: first you make an educated guess at parsing, and then you ask the user to correct it. That way you'll train the algorithm as you go.

**Comment by gnimmelf** [comment]
Good idea. Could divide it into smaller parts that work together. One for scraping,  one for translation,  one for output as a shopping list,  one for placing the order at your selected online store,  which will mark/remove ordered items from the shopping list, leaving with you the rest to go find manually.
